packages/block-editor/src/components/colors/use-colors.js

```diff
@@ -219,26 +221,10 @@ export default function __experimentalUseColors(
 		return (
 			( needsBackgroundColor || needsColor ) &&
 			withFallbackStyles(
```

Comment:
Somehow it would be good if this can be rewritten with hooks... Need to look into it.